upgrade/1.4.4/CSM-Only.md

```diff
@@ -0,0 +1,148 @@
+# CSM Only Upgrade
+
+This page provides guidance exclusively for systems upgrading CSM v1.4.X to CSM v1.4.4.
+
+The [v1.4.4](../1.4.4/README.md) upgrade page will refer to this page
+during [Update NCN images](../1.4.4/README.md#update-ncn-images).
+
+**If other products are installed on the system, return to [Update NCN images](../1.4.4/README.md#update-ncn-images) and
+choose option 2.**
+
+## Requirements
+
+* `CSM_RELEASE` is set in the shell environment on `ncn-m001`.
+
+## Steps
+
+1. (`ncn-m001#`) Generate a new CFS configuration for the management nodes.
+
+   This script creates a new CFS configuration that includes the CSM version in its name and applies it to the
+   management nodes. This leaves the management node components in CFS disabled. They will be automatically enabled when
+   they are rebooted at a later stage in the upgrade.
+
+   ```bash
+   /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
+       --no-enable --config-name management-${CSM_RELEASE}
+   ```
+
+   Successful output should end with the following line:
+
+   ```text
+   All components updated successfully.
+   ```
+
+1. Set the IMS IDs for each Management node subrole.
```

Comment:
I need a pragmatic way to get these. I know I can get the IMS ID from `kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}'` but the image name has a non-deterministic path.

```bash
ncn-m001:~ # kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' | yq4 eval '."1.4.4-beta.1".images' -
cray-shasta-csm-sles15sp4-barebones.x86_64-csm-1.4:
  id: 2648b5d9-8a49-4164-9298-125249ae7d61
secure-kubernetes-0.4.71-x86_64.squashfs:
  id: 9574bcd8-aef5-449d-858e-c0ed36a20816
secure-storage-ceph-0.3.59-x86_64.squashfs:
  id: a68077ac-7d6f-4fe3-ab6f-58f25bcff068
```

I can’t grab those without knowing the secure-kubernetes-0.4.71-x86_64.squashfs value

Reply:
Here's one way (example from mug, so version is different):

```
ncn-m001:~ # kubectl -n services get configmap cray-product-catalog -o jsonpath='{.data.csm}' | yq r -j - | jq -r '.["1.5.0-beta.22"].images | to_entries | map(select(.key | contains("secure-kubernetes")))[].value.id'
545ba744-d5e4-4725-9960-3bbe2d9be8d0
```